java/feature-collection-layer/README.md

```diff
@@ -1,26 +1,37 @@
-# Feature Collection Layer
-Creates a new Feature Collection with a Point, Polyline, and Polygon Feature Collection Table. The Collection is then displayed on the ArcGISMap as a Feature Collection Layer.
+# Feature collection layer
 
-![Feature Collection Layer App](feature-collection-layer.png)
+Create a Feature Collection Layer from a Feature Collection Table, and add it to a map.
+
+![Image of feature collection layer](feature-collection-layer.png)
+
+## Use case
+
+A Feature Collection allows easily importing external data (such as CSV files), as well as creating custom schema for data that is in non-standardized format. This data can then be used to populate a Feature Collection Table, and displayed in a Feature Collection Layer using the attributes and geometries provided in the external data source. For example, an electricity supplier could use this functionality to visualize existing location data of coverage areas (polygons), power stations (points), transmission lines (polylines), and others.
+
+## How to use the sample
+
+When launched, this sample displays a `FeatureCollectionLayer` with a `Point`, `Polyline` and `Polygon` geometry. Pan and zoom to explore the scene.
```

Comment:
picky point but we generally avoid API class references in this section. let's just convert them to plain ol' english

Reply:
okay!